### PR TITLE
Fix api docs

### PR DESCRIPTION
**Pull request by ItayGabbay**

### CODE REVIEW

File: deepchecks/vision/checks/model_evaluation/weak_segments_performance.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ class WeakSegmentsPerformance(SingleDatasetCheck, WeakSegmentAbstract):
         - ``'categorical'`` - for discrete, non-ordinal outputs. These can still be numbers,
           but these numbers do not have inherent value.
 
-        For more on image properties, see the guide about :ref:`vision__properties_guide`.
+        For more on image properties, see the guide about :ref:`vision_properties_guide`.
```
Comment (Nadav-Barak) [inline deepchecks/vision/checks/model_evaluation/weak_segments_performance.py:59]: It should be vision__properties_guide

File: deepchecks/vision/checks/train_test_validation/image_dataset_drift.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ class ImageDatasetDrift(TrainTestCheck):
         - 'categorical' - for discrete, non-ordinal outputs. These can still be numbers,
           but these numbers do not have inherent value.
 
-        For more on image / label properties, see the guide about :doc:`/user-guide/vision/vision_properties`
+        For more on image / label properties, see the guide about :doc:`/vision/usage_guides/vision_properties`
```
Comment (Nadav-Barak) [inline deepchecks/vision/checks/train_test_validation/image_dataset_drift.py:54]: ref doesnt work?

File: docs/source/checks/vision/data_integrity/plot_label_property_outliers.py [hunk 3]
```diff
@@ -39,7 +39,7 @@
 For object detection we have default built-in label properties. For other tasks you have to define your own custom label
 properties.
 For the list of the built-in object detection label properties and explanation about custom properties refer to
-:ref:`vision properties <vision__properties_guide>`.
+:ref:`vision properties <vision_properties_guide>`.
```
Comment (Nadav-Barak) [inline docs/source/checks/vision/data_integrity/plot_label_property_outliers.py:42]: same

File: docs/source/vision/tutorials/other/README.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
-.. _vision_tutorials:
+.. _vision_tutorials_other:
```
Comment (Nadav-Barak) [inline docs/source/vision/tutorials/other/README.txt:1]:
```suggestion
.. _vision__tutorials_other:
```

File: docs/source/vision/usage_guides/vision_properties.rst [hunk 5]
```diff
@@ -1,4 +1,4 @@
-.. _vision__properties_guide:
+.. _vision_properties_guide:
```
Comment (Nadav-Barak) [inline docs/source/vision/usage_guides/vision_properties.rst:1]: why?? is was same format as EVERYTHING

Comment (ItayGabbay) [reply docs/source/vision/usage_guides/vision_properties.rst:1]: It wasn't. I changed it in all relevant places.

Comment (Nadav-Barak) [reply docs/source/vision/usage_guides/vision_properties.rst:1]: in all places its the same format _tabular/vision/nlp__name_of_file

Comment (ItayGabbay) [reply docs/source/vision/usage_guides/vision_properties.rst]: There are MANY places which it's not like that. I changed what I found.

**Review by shir22 — APPROVED** [review]
🎉

**Pull request merged**